mods/ra/weapons/ballistics.yaml

```diff
@@ -118,6 +118,12 @@ TurretGun:
 	TargetActorCenter: true
 	Projectile: Bullet
 		ContrailLength: 30
+		Speed: 170
+		Inaccuracy: 1c138
+	Warhead@1Dam: SpreadDamage
+		Falloff: 100, 55, 20, 5
+		Versus:
+			None: 60
```

Comment:
Technically, `Versus: None: 60` is better moved to `^Artillery` and removed from any subsequent inheritance (except `DepthCharge` where it needs to be re-added as 90) to streamline it; there are 4 weapons that use `^Artillery` - `155mm`, `8inch`, `Grenade` and `DepthCharge`, 3 of them now have `None: 60`.

Actually, `Light` and `Heavy` are the same for `8inch` as they are for the base weapon, same for `Heavy` in `Grenade`.

Reply:
I'd prefer to keep things the way they are for this PR and release. We're already running on a much tighter timescale than I like, and it would be dangerous to introduce last minute refactorings across multiple weapons with no testing.

Reply:
Agreed. I may look into it later though if needed, am I right to assume that the preference is to have as less modifications over inherited options as possible?

Reply:
Correct.